backend/src/services/sumo_access/inplace_volumetrics_access.py

```diff
@@ -125,13 +128,50 @@ def get_table(self, table_name: str, column_name: str) -> pa.Table:
             iteration=self._iteration_name,
             column=column_name,
         )
-        if len(vol_table_collection) != 1:
+        if not vol_table_collection:
+            print(f"No aggregated volumetric tables found {self._case_uuid}, {table_name}, {column_name}")
+            print("Aggregating manually from realization tables...")
+            full_table = self.temporary_aggregate_from_realization_tables(table_name)
+            return full_table.select([column_name, "REAL", "FACIES", "ZONE", "REGION"])
+
+        if len(vol_table_collection) > 1:
             raise ValueError(f"None or multiple volumetric tables found {self._case_uuid}, {table_name}, {column_name}")
         vol_table = vol_table_collection[0]
         byte_stream: BytesIO = vol_table.blob
         table: pa.Table = pq.read_table(byte_stream)
         return table
 
+    def temporary_aggregate_from_realization_tables(self, table_name: str) -> pa.Table:
+        """Temporary function to aggregate from realization tables when no aggregated table is available
+        Assume Sumo will handle this in the future"""
+        case = self.case_collection[0]
+        vol_table_collection: TableCollection = case.tables.filter(
+            stage="realization",
+            name=table_name,
+            tagname="vol",
+            iteration=self._iteration_name,
+        )
+        if not vol_table_collection:
+            raise ValueError(f"No volumetric realization tables found {self._case_uuid}, {table_name}")
+
+        ### Using ThreadPoolExecutor to parallelize the download of the tables
+
+        def worker(idx):
+            vol_table = vol_table_collection[idx]
+            print(f"Downloading table: {table_name} for realization {vol_table.realization}")
+            byte_stream: BytesIO = vol_table.blob
+
+            table: pd.DataFrame = pd.read_csv(byte_stream)
+            table["REAL"] = vol_table.realization
+            return table
+
+        with ThreadPoolExecutor() as executor:
+            tables = list(executor.map(worker, list(range(len(vol_table_collection)))))
+            tables = pd.concat(tables)
+            tables = pa.Table.from_pandas(tables)
```

Comment:
Should we make an issue investigating if `asyncio` could/should be used instead of threads?